doc/content_item_fields.md

```diff
@@ -208,6 +222,12 @@ modified in it.
 It contains the timestamp at which the content was last modified in any way.
 This is suitable to be used for update versioning.
 
+When using the publishing api putting content, patching links and publishing
```

Comment:
I find this text confusing because we are talking about requests to Publishing API and not actions.
Can you change it for something along the following lines?

```
Performing any of the following requests to publishing api will update the timestamp:

- put content
- patch links
- publish

```